torch/fx/experimental/fuser.py

```diff
@@ -0,0 +1,45 @@
+from torch.fx import (
+    GraphModule,
+)
```

Comment:
I see there's a mypy error here. You can try 2 things to fix this:

1. `import torch.fx` and use `GraphModule` fully-qualified
2. add `__all__` to `torch/fx/__init__.py` and include all the exported names from that file. I think this is the "right" way to fix this

Reply:
I'll do the "incorrect" way for now :P 

If that's the correct way we can do it in another PR.

Reply:
Actually, that doesn't fix it lol.